packages/platform-core/API.md

```diff
@@ -30,6 +30,22 @@ export enum CDKContextKey {
     DEPLOYMENT_TYPE = "amplify-backend-type"
 }
 
+// @public
+export class ConfigController {
+    constructor(projectName?: string, configFileName?: string);
+    clear(): void;
+    // (undocumented)
+    configFilePath: string;
+    // (undocumented)
+    dirPath: string;
+    get<T>(path: string): T;
+    getConfigPath(name: string): string;
+    set(path: string, value: string | boolean | number, writeToFile?: boolean): void;
+    // (undocumented)
+    _store: Record<string, unknown>;
+    write(): void;

```

Comment:
These should be private. We should only have `get`, `set` and `clear`, everything else is internal detail.